seb_openedx/models.py

```diff
@@ -23,3 +25,35 @@ class Meta(object):
         indexes = [
             models.Index(fields=['username', 'course_id'])
         ]
+
+
+class SebCourseConfiguration(models.Model):
+    """Model that stores settings per course for seb browser."""
+    course_id = CourseKeyField(max_length=255, unique=True)
+    permission_components = models.TextField(blank=True, default='AlwaysAllowStaff,CheckSEBHashBrowserExamKey,CheckSEBHashConfigKey')
+    browser_keys = models.TextField(blank=True)
+    config_keys = models.TextField(blank=True)
+    user_banning_enabled = models.BooleanField(default=False)
+    blacklist_chapters = models.TextField(blank=True)
+    whitelist_paths = models.TextField(blank=True, default='wiki,about')
+
+    class Meta(object):
+        """Meta."""
+        app_label = 'seb_openedx'
+
+    def __unicode__(self):
+        """Formats impression of object."""
+        return six.text_type(self.course_id)
+
+    @classmethod
+    def get_config_by_course_id(cls, course_id):
+        """Get config by course_id."""
+        queryset = cls.objects.get(course_id=course_id)
```

Comment:
Having the name queryset for this object is a little confusing. I suggest to use instance.